src/backend/dx12/src/lib.rs

```diff
@@ -322,6 +322,46 @@ impl hal::PhysicalDevice<Backend> for PhysicalDevice {
         format::query_properties()[idx]
     }
 
+    fn image_format_properties(
+        &self, _format: f::Format, dimensions: u8, _tiling: image::Tiling,
```

Comment:
Should handle tiling also in case of linear tiling:
- only 2d
- single mip and array layer
- single sample count